src/components/core/constants.js

```diff
@@ -29,4 +33,42 @@ const I18N = {
     }
 }
 
-export {ROUTES, I18N} ;
+const DialogItem = props => {
+    const way = ROUTES.MESSAGES + "/" + props.id;
+  
+    return (
+      <div className={s.dialog + " " + s.active}>
+        <NavLink to={way}>{props.name}</NavLink>
+      </div>
+    );
+  };
+
+const Message = props => {
+    return <div className={s.dialogs}>{props.message}</div>;
+  };
+
+const dialogsData = [
```

Comment:
There is a convention place for data related variables - consider creating a "mocks.js" file next to the current file and keeping the data related stuff there.

Reply:
done